src/Packagist/WebBundle/Resources/views/Package/viewPackage.html.twig

```diff
@@ -10,6 +10,48 @@
     {{ parent() }}
 {% endblock %}
 
+{% block head_additions %}
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.5.0/styles/monokai_sublime.min.css">
+    <script type="text/javascript">
+      WebFontConfig = {
```

Comment:
The indent looks wrong to me.

You start with 4 spaces but use 2 spaces after that.